app-text/calibre/calibre-5.44.0-r2.ebuild

```
@@ -0,0 +1,271 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
```

3.9 is gone and a noop, so
```suggestion
PYTHON_COMPAT=( python3_{10..11} )
```